### ISSUE TITLE
Feature request: delay/disable automatic redraw

### ISSUE THREAD

**Issue by jimmykl** [abey79/vsketch#88]
I am working with the flow_img plugin which takes some time to process and would like to disable the automatic refresh until I have finished editing multiple UI parameters.

It would be ideal to have a 'Refresh' button to manually redraw when desired or even just being able to set the delay before redrawing would do the trick.

**Comment by abey79** [comment]
Makes sense for heavy sketch. I guess the best is a "auto-refresh" checkbox and a "refresh" button available when the checkbox is disabled.
